packages/manager/modules/pci/src/projects/project/storages/components/password-clipboard/controller.js

```diff
@@ -22,4 +46,25 @@ export default class PasswordClipboardComponentController {
         : 'pci_projects_project_storages_components_password_clipboard_label_show_password',
     );
   }
+
+  reset() {
+    this.model.isTextCopied = false;
+    this.model.tooltipText = this.getClipboardTooltipContent();
+  }
+
+  onClipboardClick() {
+    return this.reset();
```

Comment:
Can we set this.model directly here by avoiding reset() call.

Reply:
i think it's better to have a method for that.
we are more explicit and we know what the function do.

But you have right, a function with name reset means nothing.
i changed it to `resetToDefault()`.

Reply:
Okay Mohammed